src/Sylius/Bundle/ChannelBundle/composer.json

```diff
@@ -41,6 +43,7 @@
         "psr-4": { "Sylius\\Bundle\\ChannelBundle\\spec\\": "spec/" }
     },
     "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Installing that bundle standalone without prefer-stable downloads a lot of non-stable dependencies. If we want to test bundles standalone, I believe that it is the way to go. 

It does not affect any packages using it as these configuration options - "minimum-stability" and "prefer-stable" - are considered only for root packages. Sylius dependencies requiring `0.17.*` still resolves to the latest, development version.